test/visual/equal-width-tabs-lumo.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+  <meta charset="UTF-8">
+  <title>Equal width tabs visual tests</title>
+</head>
+
+<body>
+  <style media="screen">
+    .capture-block {
+      display: block;
```

Comment:
Nit: If not decided to change to `flex`, can be changed to `inline-block` to align with other.

Reply:
Changed to flex as you suggested.